Sources/SpeziOnboarding/ConsentView/ConsentDocumentExport.swift

```diff
@@ -0,0 +1,39 @@
+import PDFKit
+
+/// A type representing an exported `ConsentDocument`. It holds the exported `PDFDocument` and the corresponding `ConsentDocumentIdentifier`.
+public actor ConsentDocumentExport {
```

Comment:
Thank you for introducing this type here; looks nice!

As noted on the identifier type, we don't really need the String indirection for now so I would suggest to keep it simply a string for now.